README.rst

```diff
@@ -53,45 +45,66 @@ Example analysis script
    # Iterate through trajectories
    for ts in u.trajectory:
        print(ag.center_of_mass())
- 
-There are a number of tutorials_ on the MDAnalysis homepage that explain
-how to conduct RMSD calculations, Alignment and many more features of MDAnalysis.
 
-Source code
-===========
 
-Source code is hosted in a git repository at
+Documentation
+=============
+
+**New users** should read the `Quickstart Guide`_.
+
+**All users** should read the `User Guide`_.
+
+**Developers** may also want to refer to the `MDAnalysis API docs`_.
 
-https://github.com/MDAnalysis/mdanalysis
+There are a number of tutorials_ on the MDAnalysis homepage that
```

Comment:
Should we be directing users to use the tutorials on the user guide instead?

Reply:
If we do this, we might want to point the developers directly to the "Contributing" section (in addition to the API documentation)

Reply:
That is being done on lines 83-84.

Reply:
@lilyminium https://userguide.mdanalysis.org/ is 404 ... so I'll use https://www.mdanalysis.org/UserGuide/ for now and direct to https://www.mdanalysis.org/UserGuide/examples/README.html although "Examples" should perhaps be renamed to "Tutorials"?

Reply:
tutorials sends users to User Guide examples in 3787b255413bf12cbd4077ccc1d194279f27aa96